Document repo structure in README.md (#31)
lbl-cbg/t5-commons: README.md

# t5-commons
Code required for the Taskforce5 commons project. 

# Packages
* **t5common** :  A package for common operations in the Taskforce5 commons, e.g., interacting with JIRA and creating compute jobs
* **bilbomd** : Code for connecting bilbomd with Jamo and Jira
* **jamo** : Necessary files for setting up JAMO at NERSC for T5

> 

Reviewer: ```suggestion
* **jbrowse**: Code running the JBrowse genome browserfor Taskforce5
```

> * **jamo** : Necessary files for setting up JAMO at NERSC for T5

Reviewer: ```suggestion
* **dm** : Code and other files for setting up data management with JAMO for T5. This also includes everything necessary for deploying to Kubernetes.
```

Reviewer: see #37

> * **jbrowse**: Code running the JBrowse genome browserfor Taskforce5

Author: ```suggestion
* **jbrowse**: Code running the JBrowse genome browser for Taskforce5
```

> * **dm** : Code and other files for setting up data management with JAMO for T5. This also includes everything necessary for deploying to Kubernetes.

Reviewer: ```suggestion
* **data_management** : Code and other files for setting up data management with JAMO for T5. This also includes everything necessary for deploying to Kubernetes.
```

Update to reflect change: https://github.com/lbl-cbg/t5-commons/pull/37#issuecomment-2688772885

> * **jbrowse**: Code running the JBrowse genome browser for Taskforce5

Reviewer: ```suggestion
* **genome_browser**: Code running the JBrowse genome browser for Taskforce5
```
